### PR TITLE
Modification Table

### PR DESCRIPTION
**Pull request by angelique888**

### CODE REVIEW

**Review by AnneDAMIEN — APPROVED** [review]

**Review by yannduhamel — APPROVED** [review]

**Review by enerisan — DISMISSED** [review]

**Review by ioayoub — APPROVED** [review]

**Review by Pierre-Voisin — DISMISSED** [review]

**Review by Pierre-Voisin — APPROVED** [review]

**Review by yannduhamel — DISMISSED** [review]

**Review by AnneDAMIEN — APPROVED** [review]

**Review by enerisan — APPROVED** [review]

File: server/database/schema.sql [hunk 1]
```diff
@@ -1,3 +1,14 @@
+CREATE TABLE user (
```
Comment (ioayoub) [inline server/database/schema.sql:1]: - Missing constraints in this table.
- The field comment_quantity is useless here
- about the is_admin field : What if client ask for a third role tomorrow ?

File: server/database/schema.sql [hunk 2]
```diff
@@ -1,3 +1,14 @@
...
+    firstname VARCHAR(80),
+    lastname VARCHAR(80),
+    password VARCHAR(80),
+    pseudo VARCHAR(80),
+    image_profile TEXT,
+    email VARCHAR(80),
+    is_admin BOOLEAN,
+    comment_quantity INT NULL
+);
 CREATE TABLE ingredient (
```
Comment (ioayoub) [inline server/database/schema.sql:12]: Missing constraints

File: server/database/schema.sql [hunk 3]
```diff
@@ -64,7 +65,6 @@ CREATE TABLE recipe (
 CREATE TABLE comment (
```
Comment (ioayoub) [inline server/database/schema.sql:65]: Missing constraints

**Review by ioayoub — CHANGES_REQUESTED** [review]
Update your constraints on your schema.sql.
Actually too much field can have a NULL value.

**Pull request closed without merging**